src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -646,6 +646,9 @@ export class ConnectionWidget extends lifecycle.Disposable {
 				// Populate tenants (select first by default for initialization of tenant dialog)
 				await this.onAzureAccountSelected(true);
 				this._azureTenantDropdown.enable();
+				// Populate username as 'email' of selected azure account in dropdown (Ask from MySQL)
+				this._userNameInputBox.value = this._azureAccountList.find(a => a.displayInfo.displayName === this._azureAccountDropdown.value)?.displayInfo.email!
```

Comment:
Why are you defaulting to `this._azureAccountList[0]`? Not only does that not seem like not having a match from the find be something that shouldn't ever happen, if it did I'm not sure populating whatever the first value is makes sense.

Reply:
When creating a new connection, the first account in the list is pre-selected, but it's not done via 'select' on dropdown. We're picking the same value when dropdown value is not available.

Reply:
So the dropdown shows the first selected value but the actual `value` property is empty? That seems like incorrect behavior - I would expect that value always contains the currently selected (displayed) value if one exists. 

@aasimkhan30 You were doing a bunch of work around dropdown selection, do you remember if this is intended behavior?

Regardless, fixing/changing that is out of scope for this PR so could you at least add a comment explaining why you have to do this here @cheenamalhotra?

Reply:
Calling 'select' on account dropdown on load, has caused issues in the past and we fixed it with https://github.com/microsoft/azuredatastudio/pull/21749

Reply:
Well, that's not great. Seems like the real issue there was the ordering that things get selected - so not sure that the current workaround is the best option there since now we have to remember to do this special check every time we get the selected account value. Consider at the very least making a helper function to do this so the logic is at least consistent.